solidity_deposit_contract/README.md

```diff
@@ -23,3 +23,16 @@ Install the latest version of `dapp` by following the instructions at [dapp.tool
 ```sh
 make test_deposit_contract
 ```
+## Verifying bytecode
+
+The following parameters were used to generate the bytecode for the `DepositContract` available in this repository:
+
+* Contract Name: `DepositContract`
+* Compiler Version: Solidity `v0.6.11+commit.5ef660b1`
+* Optimization Enabled: `Yes` with `5000000` runs
+* EVM Version: `default` (Istanbul)
```

Comment:
@axic would it be better if we add `--evm-version istanbul`?

Reply:
It doesn't matter as it is tied to a specific compiler version, which ensures that. Would matter if it were flexible with the compiler version.

Reply:
If so, perhaps remove this line? It may look confusing when after post-Istanbul in the future.

```suggestion
```